### PR TITLE
fix: panic on node nil

### PR DESCRIPTION
**Pull request by guilhem**

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/koordlet/koordlet.go [hunk 1]
```diff
@@ -105,7 +105,13 @@ func NewDaemon(config *config.Configuration) (Daemon, error) {
 		}
 		klog.Infof("can not detect cgroup driver from 'kubepods' cgroup name")
 
-		port := int(statesInformer.GetNode().Status.DaemonEndpoints.KubeletEndpoint.Port)
+		node := statesInformer.GetNode()
```
Comment (eahydra) [inline pkg/koordlet/koordlet.go:108]: thanks for your fix the issue, but why can't get Node, have you found the root cause?

Comment (guilhem) [reply pkg/koordlet/koordlet.go:108]: Root cause seems to be #407

**Pull request merged**